guideforms.py

```diff
@@ -0,0 +1,308 @@
+         'What metrics/measurement/feedback will you be using to validate '
+         'designs? Double check that your experiment makes sense given that '
+         'a large developer (e.g. a Google product or Facebook) likely '
+         'can\'t use it in production due to the limits enforced by origin '
+         'trials.\n\nIf Intent to Extend Origin Trial, highlight new/different '
+         'areas for experimentation. Should not be an exact copy of goals '
+         'from the first Intent to Experiment.')),
+
+    'experiment_timeline': forms.CharField(
+        label='Experiment Timeline', required=False,
+        widget=forms.Textarea(attrs={'rows': 2, 'cols': 50, 'maxlength': 1480}),
+        help_text='When does the experiment start and expire?'),
+
+    'experiment_risks': forms.CharField(
+        label='Experiment Risks', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
+        help_text=
+        ('When this experiment comes to an end are there any risks to the '
+         'sites that were using it, for example losing access to important '
+         'storage due to an experimental storage API?')),
+
+    'experiment_extension_reason': forms.CharField(
+        label='Experiment Extension Reason', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
+        help_text=
+        ('If this is a repeat experiment, link to the previous Intent to '
+         'Experiment thread and explain why you want to extend this '
+         'experiment.')),
+
+    'ongoing_constraints': forms.CharField(
+        label='Ongoing Constraints', required=False,
+        widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
+        help_text=
+        ('Do you anticipate adding any ongoing technical constraints to '
+         'the codebase while implementing this feature? We prefer to avoid '
+         'features which require or assume a specific architecture. '
+         'For most features, the answer here is "None."')),
+
+    }
+
+
+class OverviewForm(forms.Form):
+
+  name = ALL_FIELDS['name']
+  summary = ALL_FIELDS['summary']
+  category = ALL_FIELDS['category']
+
+
+class Incubate(forms.Form):
+
+  current_user_email = users.get_current_user().email if users.get_current_user() else None
+  owner = forms.CharField(
+      initial=current_user_email, required=True, label='Contact emails',
+      help_text=('Comma separated list of full email addresses. '
+                 'Prefer @chromium.org.'))
+
+  motivation = ALL_FIELDS['motivation']
+  doc_links = ALL_FIELDS['doc_links']
+  standardization = ALL_FIELDS['standardization']
+  spec_link = ALL_FIELDS['spec_link']
+
+
+class Prototype(forms.Form):
+
+  doc_links = ALL_FIELDS['doc_links']
+  standardization = ALL_FIELDS['standardization']
+  spec_link = ALL_FIELDS['spec_link']
+  tag_review = ALL_FIELDS['tag_review']
+  intent_to_implement_url = ALL_FIELDS['intent_to_implement_url']
+
+
+class DevTrial(forms.Form):
+
```

Comment:
Why are there no begin and milestone files?

Reply:
This CL only has 4 process phases out of the 9 in Chris's spreadsheet.  I'm planning on getting that much working before going for completeness.  I have added a TODO to make sure that every field in the old form is included.